src/Extensions/Abstractions/IMachineInformation.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Net;
+
+namespace Microsoft.Omex.Extensions.Abstractions
```

Comment:
>Extensions [](start = 25, length = 10)

Is the extensions method appropriate in this context? It can be confusing with C# extensions.

Reply:
I've selected this name since all of the created projects are implement interfaces from:
`Microsoft.Extensions.Loggers.Abstraction`
`Microsoft.Extensions.Hosting.Abstraction`
And `Microsoft.Omex.Extensions.Abstraction` is Omex interfaces and classes that need to be shared between projects.